libtsuba/src/RDG.cpp

```diff
@@ -742,6 +951,58 @@ tsuba::RDG::set_part_metadata(const tsuba::PartitionMetadata& metadata) {
   core_->part_header().set_metadata(metadata);
 }
 
+tsuba::EntityTypeIDPairRange
+tsuba::RDG::node_entity_type_id_dictionary() const {
+  return FlattenEntityTypeIDToSetOfEntitySetIDsMap(
+      core_->part_header().node_entity_type_id_dictionary());
+}
+
+void
+tsuba::RDG::set_node_entity_type_id_dictionary(
+    EntityTypeIDPairRange node_entity_type_id_dictionary_range) {
+  core_->part_header().set_node_entity_type_id_dictionary(
+      InflateEntityTypeIDPairRange(node_entity_type_id_dictionary_range));
+}
+
+tsuba::EntityTypeIDPairRange
+tsuba::RDG::edge_entity_type_id_dictionary() const {
+  return FlattenEntityTypeIDToSetOfEntitySetIDsMap(
```

Comment:
Where are these inflate and Flatten methods defined?

Reply:
RDG.h line 367